### PR TITLE
added angular 15 upgrade guide

### PR DESCRIPTION
**Pull request by DawidSAG**

### CODE REVIEW

File: content/web/upgrade-bundle/upgrading-to-angular-15.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# layout: redirect
+weight: 6
+---
+
+Angular 15 is supported from version `1018.157.0`. The following configuration changes are required before you can run the application:
+
+- Update all `@angular/*` dependencies to `15.2.7`.
+- Update `TypeScript` to version `4.9.5`.
+- Follow the `Angular 15` upgrade guide: [Updating to version 14](https://update.angular.io/?l=3&v=14.0-15.0).
+- Use Node version `14.20.x`.
```
Comment (janhommes) [inline content/web/upgrade-bundle/upgrading-to-angular-15.md:12]: Is this the minimum version? 16 would also work and we use it mostly. I guess 14 is deprecated already.

File: content/web/upgrade-bundle/upgrading-to-angular-15.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title: Upgrading from Angular 14 to Angular 15
+# layout: redirect
+weight: 6
+---
+
+Angular 15 is supported from version `1018.157.0`. The following configuration changes are required before you can run the application:
+
+- Update all `@angular/*` dependencies to `15.2.7`.
+- Update `TypeScript` to version `4.9.5`.
+- Follow the `Angular 15` upgrade guide: [Updating to version 14](https://update.angular.io/?l=3&v=14.0-15.0).
```
Comment (janhommes) [inline content/web/upgrade-bundle/upgrading-to-angular-15.md:11]:
```suggestion
- Follow the `Angular 15` upgrade guide: [Updating to version 15](https://update.angular.io/?l=3&v=14.0-15.0).
```

File: content/web/upgrade-bundle/upgrading-to-angular-15.md (hunk 1 above)
Comment (DawidSAG) [reply content/web/upgrade-bundle/upgrading-to-angular-15.md:12]: I can align it with the Angular guide:

"Make sure that you are using a supported version of node.js before you upgrade your application. Angular v15 supports node.js versions: 14.20.x, 16.13.x and 18.10.x. [Read further](https://angular.io/guide/update-to-version-15#v15-bc-01)"

File: content/web/upgrade-bundle/upgrading-to-angular-15.md (hunk 2 above)
Comment (DawidSAG) [reply content/web/upgrade-bundle/upgrading-to-angular-15.md:11]: done

File: content/web/upgrade-bundle/upgrading-to-angular-15.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+For more information on multi provider, go to this section: [Multi provider (MP)](/web/libraries/#multi-provider-mp)
+{{< /c8y-admon-info >}}
+
+
+{{< c8y-admon-info >}}
+AOT (Ahead-of-time compilation) is not yet supported.
+{{< /c8y-admon-info >}}
+
+
+{{< c8y-admon-info >}}
```
Comment (janhommes) [inline content/web/upgrade-bundle/upgrading-to-angular-15.md:40]: #trivial guess we can remove this now, or?

**Review by janhommes — APPROVED** [review]

**Comment by MWindel** [comment]
@DawidSAG Does this need to go into other release versions? If so, please do the changes in a separate PR.

File: content/web/upgrade-bundle/upgrading-to-angular-15.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Angular 15 is supported from version `1018.157.0`. The following configuration changes are required before you can run the application:
+
+- Update all `@angular/*` dependencies to `15.2.7`.
+- Update `TypeScript` to version `4.9.5`.
+- Follow the `Angular 15` upgrade guide: [Updating to version 15](https://update.angular.io/?l=3&v=14.0-15.0).
+- Use Node version `^14.20.0 || ^16.13.0 || ^18.10.0`.
+- Delete `node_modules` and reinstall them.
+
+- Token hooks were deprecated and replaced by function hooks. Check the table below to see to which function hooks the token hooks have been migrated.
```
[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/web/upgrade-bundle/upgrading-to-angular-15.md:15]: No token hooks are migrated to the function hooks, i.e. `HOOK_TABS` should not be used anymore, developers should use `hookTab` instead.

**Comment by DawidSAG** [comment]
> @DawidSAG Does this need to go into other release versions? If so, please do the changes in a separate PR.

No, we don't need it in other versions.

File: content/web/upgrade-bundle/upgrading-to-angular-15.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (MWindel) [reply content/web/upgrade-bundle/upgrading-to-angular-15.md:15]: @DawidSAG This sentence does not make much sense to me:
"Check the table below to see to which function hooks the token hooks have been migrated." Could you elaborate?

Is it "Check the table below to see to which function hooks the token hooks have been migrated _to_"?

[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/web/upgrade-bundle/upgrading-to-angular-15.md:15]: @MWindel yes, you're right.

[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/web/upgrade-bundle/upgrading-to-angular-15.md]: updated

**Review by MWindel — APPROVED** [review]

**Pull request merged**